tests/functional/api-suite-info/00-get-graph-raw-1/suite.rc

```diff
@@ -4,7 +4,10 @@
 [runtime]
     [[t1]]
         script = """
-ctb-get-graph-raw 'start_point_string=1' 'stop_point_string=1' 'group_nodes=T' \
+# Use the same Python (and lib) as the Cylc version being run.
+PYTHON=$(head -n 1 $(which cylc))
+${PYTHON#??} ${CYLC_SUITE_DEF_PATH}/bin/ctb-get-graph-raw \
```

Comment:
```suggestion
python3 ${CYLC_SUITE_DEF_PATH}/bin/ctb-get-graph-raw \
```
So, after a little debugging with @oliver-sanders, we have established that this does not work with met office conda environments and will fail for us.  
Since each of these tests run local background jobs, this fix should be grand.

Reply:
Yeah, this wasn't a smart change on my part - it works only for Python virtual envs (which replace the shebang line in installed scripts with the path the env's Python interpreter) and only then if there's no `cylc` wrapper preprended to PATH in the user's `.bashrc`.

Reply:
I guess your suggestion should work because the correct environment has to be activated for the job (whether via the wrapper or by inheriting from the scheduler environment) and in that environment `python3` will be that environment-specific Python version... yep (tested) :+1: